plotly/src/plot.rs

```diff
@@ -29,6 +38,17 @@ struct StaticPlotTemplate<'a> {
     height: usize,
 }
 
+#[cfg(feature = "plotly_noembed")]
+#[derive(Template)]
+#[template(path = "static_plot_noembed.html", escape = "none")]
+#[cfg(not(target_family = "wasm"))]
+struct StaticPlotTemplate<'a> {
+    plot: &'a Plot,
+    format: ImageFormat,
+    width: usize,
+    height: usize,
+}
```

Comment:
Not sure if there is a way to do this without having to duplicate the whole struct.